presentation/tutorial.jl

```diff
@@ -0,0 +1,815 @@
+### A Pluto.jl notebook ###
+# v0.17.7
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 19b71044-a0c8-468a-8e13-91b17d336497
+import Pkg; Pkg.activate
+
+# ╔═╡ 599711e7-ee51-4ca0-94ac-8c1a67f6ac8d
+using CSV, Microbiome, Microbiome.SparseArrays, BiobakeryUtils, BiobakeryUtils.Conda, Random;
+
+# ╔═╡ 84470294-5a17-469e-890a-5e2b9f777dbb
+using Downloads: download
+
+# ╔═╡ 161527c0-d2c9-4ef8-a223-4946b5f0e083
+# Installing packages
```

Comment:
instead of commenting these lines out, you can make these markdown headings with the code format below.
```
md"""
# Installing packages
"""
```

then you can hide the code in the Pluto notebook and only show the result, and you get a nice heading format